blog/_posts/2020-07-5-june-commit-of-the-month.md

```diff
@@ -16,9 +16,21 @@ keywords:
   - tilt
 ---
 
-# Outline
+June's commit of the month is [0860db](https://github.com/tilt-dev/tilt/commit/0860db76f8cbaa7dae551f7800f5480593a2ec95)! To explain why let's start with a bit of background on Tilt Extensions.
 
-https://github.com/tilt-dev/tilt/commit/0860db76f8cbaa7dae551f7800f5480593a2ec95
+[Extensions were released](2020/04/01/more-customizable-tiltfiles-with-extensions.html) a couple months ago to make it easier to share Tiltfile functionality with other Tilt users. You can think of Tilt Extensions as a Tiltfile package manager.
+
+You can see all of the currently available Tilt Extensions in the [Tilt Extensions Repository](https://github.com/tilt-dev/tilt-extensions). More than 10 extensions! Some of them, like the [`git_resource`](https://github.com/tilt-dev/tilt-extensions/tree/master/git_resource) extension, already have multiple contributors on their own.
+
+It's natural that as this ecosystem grows extension authors would want to use extensions from other extensions. For example, many extensions could benefit from using the [`local_output` extension](https://github.com/tilt-dev/tilt-extensions/tree/master/local_output). Unfortunately we made the decision early on to disallow loading extensions from other extensions in order to avoid having to deal with the [diamond dependency](https://www.well-typed.com/blog/2008/04/the-dreaded-diamond-dependency-problem/) problem. The diamond dependency problem is essentially: what happens if packages transitively depend on different versions of the same package. For example:
```

Comment:
```suggestion
It's natural that as this ecosystem grows, extension authors would want to use others' extensions in their own. For example, many extensions could benefit from using the [`local_output` extension](https://github.com/tilt-dev/tilt-extensions/tree/master/local_output). Unfortunately we made the decision early on to disallow loading extensions from other extensions in order to avoid having to deal with the [diamond dependency problem](https://www.well-typed.com/blog/2008/04/the-dreaded-diamond-dependency-problem/). The diamond dependency problem is essentially: what happens if packages transitively depend on different versions of the same package? For example:
```